Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs

```diff
@@ -16,7 +16,7 @@ public ShopLensSpeechRecognizer(Action<object, ShopLensSpeechRecognizedEventArgs
         public void RecognizePhrase(Activity activity)
         {
             // According to Java, Voice Recognizer needs to be run on the Main Ui Thread.
-            activity.RunOnUiThread(ListenForAPhrase);
+            activity?.RunOnUiThread(ListenForAPhrase);
```

Comment:
When can activity be null?

Reply:
When an activity is destroyed, but a class had a reference to it and passes that reference to this method, which results in null being passed.